### PR TITLE
ref: Convert organizationMemberDetail.spec.jsx to typescript

### PR DESCRIPTION
**Pull request by ryan953**

### CODE REVIEW

File: static/app/views/settings/organizationMembers/organizationMemberDetail.spec.tsx [hunk 1]
```diff
@@ -160,16 +157,21 @@ describe('OrganizationMemberDetail', function () {
     });
 
     it('changes org role to owner', async function () {
-      render(<OrganizationMemberDetail params={{memberId: member.id}} />, {
-        context: routerContext,
-      });
+      const {routerContext, routerProps} = initializeOrg({organization});
```
Comment (evanpurkhiser) [inline static/app/views/settings/organizationMembers/organizationMemberDetail.spec.tsx:160]: I think you can just inline the org overrides here

```tsx
initalizeOrg({
  organization: {teams, features: ['team-roles']}
});
```

Comment (evanpurkhiser) [reply static/app/views/settings/organizationMembers/organizationMemberDetail.spec.tsx:160]: Do we actually need to call `initializeOrg` in every test?

Comment (evanpurkhiser) [reply static/app/views/settings/organizationMembers/organizationMemberDetail.spec.tsx:160]: Ah yeah some tests change it, cool cool cool

Comment (ryan953) [reply static/app/views/settings/organizationMembers/organizationMemberDetail.spec.tsx:160]: ya they do :( 

i started with some global consts but then eslint said no

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**